lnwire/channel_update.go

```diff
@@ -12,6 +13,11 @@ import (
 // present in the ChannelUpdate.
 type ChanUpdateMsgFlags uint8
 
+// String returns the bitfield flags as a string.
+func (c ChanUpdateMsgFlags) String() string {
+	return fmt.Sprintf("%08b", c)
```

Comment:
Nice didn't even know this directive was a thing!

Reply:
Alternatively, [we can use something like this here](https://github.com/lightningnetwork/lnd/pull/2371/commits/d6ce3bf4531fd01063ef4ebad65aa772fcda6e27). This route makes the logs even more readable IMO. Not a blocker though, we can make an issue to make the logs prettier after this lands.

Reply:
Hm yeah, could be a nice follow-up :)